### ISSUE TITLE
Cannot install Chatterbot 

### ISSUE THREAD

**Issue by z3r0memory** [gunthercox/ChatterBot#2168]
Hello!
I'm running on Windows 10 x64 with Python 3.9.4, but i can't install Chatterbot
Here's pip command output:

```
  Installing build dependencies ... error
  ERROR: Command errored out with exit status 1:
   command: 'c:\users\radeber\appdata\local\programs\python\python39\python.exe' 'c:\users\radeber\appdata\local\programs\python\python39\lib\site-packages\pip' install --ignore-installed --no-user --prefix 'C:\Users\radeber\AppData\Local\Temp\pip-build-env-6h8_mxzq\overlay' --no-warn-script-location --no-binary :none: --only-binary :none: -i https://pypi.org/simple -- setuptools 'wheel>0.32.0,<0.33.0' Cython 'cymem>=2.0.2,<2.1.0' 'preshed>=2.0.1,<2.1.0' 'murmurhash>=0.28.0,<1.1.0' 'thinc>=7.0.8,<7.1.0'
       cwd: None
  Complete output (81 lines):
  Collecting setuptools
    Using cached setuptools-57.0.0-py3-none-any.whl (821 kB)
  Collecting wheel<0.33.0,>0.32.0
    Using cached wheel-0.32.3-py2.py3-none-any.whl (21 kB)
  Collecting Cython
    Using cached Cython-0.29.23-cp39-cp39-win_amd64.whl (1.7 MB)
  Collecting cymem<2.1.0,>=2.0.2
    Using cached cymem-2.0.5-cp39-cp39-win_amd64.whl (36 kB)
  Collecting preshed<2.1.0,>=2.0.1
    Using cached preshed-2.0.1.tar.gz (113 kB)
  Collecting murmurhash<1.1.0,>=0.28.0
    Using cached murmurhash-1.0.5-cp39-cp39-win_amd64.whl (21 kB)
  Collecting thinc<7.1.0,>=7.0.8
    Using cached thinc-7.0.8.tar.gz (1.9 MB)
  Collecting blis<0.3.0,>=0.2.1
    Using cached blis-0.2.4.tar.gz (1.5 MB)
  Collecting wasabi<1.1.0,>=0.0.9
    Using cached wasabi-0.8.2-py3-none-any.whl (23 kB)
  Collecting srsly<1.1.0,>=0.0.6
    Using cached srsly-1.0.5-cp39-cp39-win_amd64.whl (177 kB)
  Collecting numpy>=1.7.0
    Using cached numpy-1.20.3-cp39-cp39-win_amd64.whl (13.7 MB)
  Collecting plac<1.0.0,>=0.9.6
    Using cached plac-0.9.6-py2.py3-none-any.whl (20 kB)
  Collecting tqdm<5.0.0,>=4.10.0
    Using cached tqdm-4.61.0-py2.py3-none-any.whl (75 kB)
  Using legacy 'setup.py install' for preshed, since package 'wheel' is not installed.
  Using legacy 'setup.py install' for thinc, since package 'wheel' is not installed.
  Using legacy 'setup.py install' for blis, since package 'wheel' is not installed.
  Installing collected packages: setuptools, wheel, Cython, cymem, preshed, murmurhash, numpy, blis, wasabi, srsly, plac, tqdm, thinc
      Running setup.py install for preshed: started
      Running setup.py install for preshed: finished with status 'error'
      ERROR: Command errored out with exit status 1:
       command: 'c:\users\radeber\appdata\local\programs\python\python39\python.exe' -u -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'C:\\Users\\radeber\\AppData\\Local\\Temp\\pip-install-06973el7\\preshed\\setup.py'"'"'; __file__='"'"'C:\\Users\\radeber\\AppData\\Local\\Temp\\pip-install-06973el7\\preshed\\setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' install --record 'C:\Users\radeber\AppData\Local\Temp\pip-record-l_p__yy3\install-record.txt' --single-version-externally-managed --prefix 'C:\Users\radeber\AppData\Local\Temp\pip-build-env-6h8_mxzq\overlay' --compile --install-headers 'C:\Users\radeber\AppData\Local\Temp\pip-build-env-6h8_mxzq\overlay\Include\preshed'
           cwd: C:\Users\radeber\AppData\Local\Temp\pip-install-06973el7\preshed\
      Complete output (41 lines):
      WARNING: The wheel package is not available.
      running install
      running build
      running build_py
      creating build
      creating build\lib.win-amd64-3.9
      creating build\lib.win-amd64-3.9\preshed
      copying preshed\about.py -> build\lib.win-amd64-3.9\preshed
      copying preshed\__init__.py -> build\lib.win-amd64-3.9\preshed
      creating build\lib.win-amd64-3.9\preshed\tests
      copying preshed\tests\test_counter.py -> build\lib.win-amd64-3.9\preshed\tests
      copying preshed\tests\test_hashing.py -> build\lib.win-amd64-3.9\preshed\tests
      copying preshed\tests\test_pop.py -> build\lib.win-amd64-3.9\preshed\tests
      copying preshed\tests\__init__.py -> build\lib.win-amd64-3.9\preshed\tests
      copying preshed\counter.pyx -> build\lib.win-amd64-3.9\preshed
      copying preshed\maps.pyx -> build\lib.win-amd64-3.9\preshed
      copying preshed\counter.pxd -> build\lib.win-amd64-3.9\preshed
      copying preshed\maps.pxd -> build\lib.win-amd64-3.9\preshed
      copying preshed\__init__.pxd -> build\lib.win-amd64-3.9\preshed
      running build_ext
      building 'preshed.maps' extension
      creating build\temp.win-amd64-3.9
      creating build\temp.win-amd64-3.9\Release
      creating build\temp.win-amd64-3.9\Release\preshed
      C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Tools\MSVC\14.28.29333\bin\HostX86\x64\cl.exe /c /nologo /Ox /W3 /GL /DNDEBUG /MD -Ic:\users\radeber\appdata\local\programs\python\python39\include -Ic:\users\radeber\appdata\local\programs\python\python39\include -Ic:\users\radeber\appdata\local\programs\python\python39\include -IC:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Tools\MSVC\14.28.29333\ATLMFC\include -IC:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Tools\MSVC\14.28.29333\include -IC:\Program Files (x86)\Windows Kits\NETFXSDK\4.8\include\um -IC:\Program Files (x86)\Windows Kits\10\include\10.0.18362.0\ucrt -IC:\Program Files (x86)\Windows Kits\10\include\10.0.18362.0\shared -IC:\Program Files (x86)\Windows Kits\10\include\10.0.18362.0\um -IC:\Program Files (x86)\Windows Kits\10\include\10.0.18362.0\winrt -IC:\Program Files (x86)\Windows Kits\10\include\10.0.18362.0\cppwinrt /EHsc /Tppreshed/maps.cpp /Fobuild\temp.win-amd64-3.9\Release\preshed/maps.obj /Ox /EHsc
      maps.cpp
      preshed/maps.cpp(5728): error C2039: "tp_print": ­Ґ пў«пҐвбп з«Ґ­®¬ "_typeobject".
      c:\users\radeber\appdata\local\programs\python\python39\include\cpython/object.h(193): note:  б¬. ®Ўкпў«Ґ­ЁҐ "_typeobject"
      preshed/maps.cpp(5740): error C2039: "tp_print": ­Ґ пў«пҐвбп з«Ґ­®¬ "_typeobject".
      c:\users\radeber\appdata\local\programs\python\python39\include\cpython/object.h(193): note:  б¬. ®Ўкпў«Ґ­ЁҐ "_typeobject"
      preshed/maps.cpp(5749): error C2039: "tp_print": ­Ґ пў«пҐвбп з«Ґ­®¬ "_typeobject".
      c:\users\radeber\appdata\local\programs\python\python39\include\cpython/object.h(193): note:  б¬. ®Ўкпў«Ґ­ЁҐ "_typeobject"
      preshed/maps.cpp(5755): error C2039: "tp_print": ­Ґ пў«пҐвбп з«Ґ­®¬ "_typeobject".
      c:\users\radeber\appdata\local\programs\python\python39\include\cpython/object.h(193): note:  б¬. ®Ўкпў«Ґ­ЁҐ "_typeobject"
      preshed/maps.cpp(5761): error C2039: "tp_print": ­Ґ пў«пҐвбп з«Ґ­®¬ "_typeobject".
      c:\users\radeber\appdata\local\programs\python\python39\include\cpython/object.h(193): note:  б¬. ®Ўкпў«Ґ­ЁҐ "_typeobject"
      preshed/maps.cpp(5767): error C2039: "tp_print": ­Ґ пў«пҐвбп з«Ґ­®¬ "_typeobject".
      c:\users\radeber\appdata\local\programs\python\python39\include\cpython/object.h(193): note:  б¬. ®Ўкпў«Ґ­ЁҐ "_typeobject"
      preshed/maps.cpp(6129): warning C4996: '_PyUnicode_get_wstr_length': deprecated in 3.3
      preshed/maps.cpp(6145): warning C4996: '_PyUnicode_get_wstr_length': deprecated in 3.3
      error: command 'C:\\Program Files (x86)\\Microsoft Visual Studio\\2019\\Community\\VC\\Tools\\MSVC\\14.28.29333\\bin\\HostX86\\x64\\cl.exe' failed with exit code 2
      ----------------------------------------
  ERROR: Command errored out with exit status 1: 'c:\users\radeber\appdata\local\programs\python\python39\python.exe' -u -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'C:\\Users\\radeber\\AppData\\Local\\Temp\\pip-install-06973el7\\preshed\\setup.py'"'"'; __file__='"'"'C:\\Users\\radeber\\AppData\\Local\\Temp\\pip-install-06973el7\\preshed\\setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' install --record 'C:\Users\radeber\AppData\Local\Temp\pip-record-l_p__yy3\install-record.txt' --single-version-externally-managed --prefix 'C:\Users\radeber\AppData\Local\Temp\pip-build-env-6h8_mxzq\overlay' --compile --install-headers 'C:\Users\radeber\AppData\Local\Temp\pip-build-env-6h8_mxzq\overlay\Include\preshed' Check the logs for full command output.
  WARNING: You are using pip version 20.2.3; however, version 21.1.2 is available.
  You should consider upgrading via the 'c:\users\radeber\appdata\local\programs\python\python39\python.exe -m pip install --upgrade pip' command.
  ----------------------------------------
ERROR: Command errored out with exit status 1: 'c:\users\radeber\appdata\local\programs\python\python39\python.exe' 'c:\users\radeber\appdata\local\programs\python\python39\lib\site-packages\pip' install --ignore-installed --no-user --prefix 'C:\Users\radeber\AppData\Local\Temp\pip-build-env-6h8_mxzq\overlay' --no-warn-script-location --no-binary :none: --only-binary :none: -i https://pypi.org/simple -- setuptools 'wheel>0.32.0,<0.33.0' Cython 'cymem>=2.0.2,<2.1.0' 'preshed>=2.0.1,<2.1.0' 'murmurhash>=0.28.0,<1.1.0' 'thinc>=7.0.8,<7.1.0' Check the logs for full command output.
```

**Comment by AdhTri-001** [comment]
Yeah same plz help

**Comment by z3r0memory** [comment]
Fixed by installing Python 3.7
